maquetado/scss/styles.scss

```diff
@@ -27,60 +27,72 @@ body {
 .main-content {
 	display: flex;
 	flex-direction: column;
-	align-items: center;
-	padding: 150px 100px;
+	padding: 50px 100px;
+	max-width: 1500px;
+	margin: 0 auto;
 
 	.title {
-		display: flex;
-		align-self: flex-start;
-		position: relative;
-		left: 0;
-		margin: 30px 40px;
-
+		margin: 120px 0 100px 60px;
+	
 		.section-title {
 			font-size: $font-big;
-			
-			&::before{
-				content: '';
-				height: $triangle-height;
-				width: $triangle-width;
-				background: url('assets/triangle.svg') no-repeat;
-				position: absolute;
-				left: -50px;
-				top: -30px;
-				z-index: -1;
-			}	
+			position: relative;
+			top: 30px;
+			left: 60px;
+		}
+		
+		&::before {
+		    background-image: url('assets/triangle.svg') ;
+		    background-size: $triangle-height 90px;
+		    display: inline-block;
+		    width: $triangle-width; 
+		    height: $triangle-height;
+		    content:"";
+		    position: absolute;
 		}
 	}
 
+
 	.techs {
-		display: flex;
-		flex-wrap: wrap;
-		justify-content: space-between;
-		padding: 30px 50px;
+		width: 100%;
+		display: grid;
+		grid-template-columns: 450px 200px auto;
+		grid-template-rows: repeat(3, max-content);
+		grid-column-gap: 40px;
 
 		.tech-card {
 			display: flex;
 			flex-direction: column;
 			align-items: center;
-			margin: 20px;
+			margin: 10px;
 			padding: 20px 20px;
-			border: 1px solid $green;
 			border-radius: 5px;
-			width: 230px;
+			width: 225px;
 			text-align: center;
+			line-height: 30px;
 			position: relative;
-			background-color: $green;
-
-			&::before {
-				content: '';
-				border-radius: 5px;
-				position: absolute;
-				top: -10px;
-				left: -10px;
-				width: 230px;
-				height: 300px;
-				border: 1px solid $green;
+
+			&.first-card {
+				grid-column: 1 / 2;
+				grid-row: 1 / 4;
+				justify-self: end;
+				align-self: center;
+				border-right: 2px solid $green;
+				height: 100%;
+				display: flex;
+				justify-content: center;
+			}
+
+			&.col2 {
```

Comment:
i believe this is no longer needed.. if you need to add the styles ```justify-self: start;``` ```align-self: center``` you can put it in one class and set the class to all the elements